### PR TITLE
time & weather: fix inconsistency

### PR DESCRIPTION
**Pull request by hugwalk**

### CODE REVIEW

**Comment by raphielscape** [comment]
Seems fine for me, lint this and I'll merge

**Comment by RealAkito** [comment]
Lint ftw

File: userbot/modules/weather.py [hunk 1]
```diff
@@ -41,16 +41,15 @@
         return
 
     if len(OWM_API) < 1:
-        await weather.edit("Get an API key from "
-                           "https://openweathermap.org/ first.")
+        await weather.edit("`Get an API key from` https://openweathermap.org/ `first.`")
```
Comment (RealAkito) [inline userbot/modules/weather.py]: No need to even edit this line...

File: userbot/modules/weather.py [hunk 2]
```diff
@@ -77,7 +76,7 @@
     result = json.loads(request.text)
 
     if request.status_code != 200:
-        await weather.edit(f"{result['message']}")
+        await weather.edit(f"`Invalid country.`")
```
Comment (RealAkito) [inline userbot/modules/weather.py]: We have it there to tell what msg the page give out alongside 200 error.

File: userbot/modules/weather.py [hunk 3]
```diff
@@ -139,7 +138,7 @@ def sun(unix):
         return
 
     if len(OWM_API) < 1:
-        await city.edit("Get an API key from https://openweathermap.org/ first.")
+        await city.edit("`Get an API key from` https://openweathermap.org/ `first.`")
```
Comment (RealAkito) [inline userbot/modules/weather.py]: Lint this.

File: userbot/modules/weather.py [hunk 4]
```diff
@@ -175,7 +174,7 @@ def sun(unix):
     result = json.loads(request.text)
 
     if request.status_code != 200:
-        await city.edit(f"{result['message']}")
+        await city.edit(f"`Invalid country.`")
```
Comment (RealAkito) [inline userbot/modules/weather.py]: We have it there to tell what msg the page give out alongside 200 error.

**Pull request merged**